test/arithmetic.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
It's best practice to name this module ```arithmetic-test.js``` in the future, and any other test file will follow the same pattern of ```foobar-test.js```.